server/views/pages/visitsByDate/visitsByDate.njk

```diff
@@ -13,10 +13,92 @@
   {{ breadcrumbs() }}
 {% endblock %}
 
+{% macro renderVisitsSection(type) %}
```

Comment:
Because the page can now have multiple tables of visits (e.g. open/closed), moved the visit table code to this macro to avoid duplication.